doc/sphinxman/source/code_style.rst

```diff
@@ -0,0 +1,100 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2017 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. _`sec:code_style`:
+
+Code style conventions
+======================
+
+It is important to keep a consistent formatting of the C++ and Python code
+to avoid hard-to-read diffs and merge conflicts.
+``clang-format``_ and ``yapf``_ can be used to format C++ and Python code,
+respectively, according to a predefined style file.
+|PSIfour| provides the :source:`.clang-format` and :source:`.style.yapf` files in the root
+folder of the project.
+It is **recommended** that modifications and/or new files checked into the
+repository are formatted according to these style files using ``clang-format``
+and ``yapf``. It is then helpful if these tools be part of your development toolchain.
+Once ``clang-format`` and ``yapf`` are installed, then there are three ways in which
+formatting of the code can be accomplished, in decreasing order of automation:
+
+1. By integrating the formatters into your editor.
+2. By installing Git hooks to run the formatters when committing.
+3. By running the formatters manually on the modified files.
+
+.. _`faq:editor`:
```

Comment:
Also go ahead and add the tags `faq:editor`, etc. to the prog-faq.rst file, I think.